charts/fybrik/values.yaml

```diff
@@ -215,8 +215,7 @@ manager:
   solver:
     # image of the container with solver binary and libs
     # when specified, the solver will be deployed in the manager pod
-    image: "optimizer"
-    imagePullPolicy: "Always"
+    image: "ghcr.io/fybrik/optimizer:master"
```

Comment:
when creating a new release for Fybrik the optimizer tag should use the release tag?

Reply:
We are not creating a new image for optimizer for each release. But maybe it should represent the tool and the version underneath.

Reply:
It might be better to use a fixed version for the optimizer rather than master for debugging purposes.

Reply:
Ok, using optimizer:0.1.